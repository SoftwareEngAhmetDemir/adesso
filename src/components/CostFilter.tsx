import CostFilterBody from "../utils/CostFilterBody";

function CostFilter() {
  
  return (
    <div>
      <div>
        <h5>Costs</h5>
      </div>
    <CostFilterBody/>
    </div>
  );
}

export default CostFilter;
