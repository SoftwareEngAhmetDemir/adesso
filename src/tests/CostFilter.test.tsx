import { shallow } from 'enzyme';
import CostFilter from '../components/CostFilter';

describe('whole CostFilter', () => {
  test('should test CostFilter component', () => {
    const wrapper = shallow(<CostFilter />);
    expect(wrapper).toMatchSnapshot();
  });
});
