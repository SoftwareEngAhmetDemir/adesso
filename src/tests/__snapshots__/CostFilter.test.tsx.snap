// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`whole CostFilter should test CostFilter component 1`] = `
<div>
  <div>
    <h5>
      Costs
    </h5>
  </div>
  <CostFilterBody />
</div>
`;
